shared/src/commonMain/kotlin/vn/luongvo/kmm/survey/data/local/datasource/TokenLocalDataSource.kt

```diff
@@ -0,0 +1,37 @@
+package vn.luongvo.kmm.survey.data.local.datasource
+
+import com.russhwolf.settings.*
+import vn.luongvo.kmm.survey.domain.model.Token
+
+private const val TOKEN_TYPE_KEY = "tokenType"
+private const val ACCESS_TOKEN_KEY = "accessToken"
+private const val REFRESH_TOKEN_KEY = "refreshToken"
+
+interface TokenLocalDataSource {
+
+    fun saveToken(token: Token)
```

Comment:
@luongvo I think this function will be called in somewhere else, not in VM so we don't need a use case for it, right? 🤔

Reply:
@manh-t this `saveToken` is being called in `LogInUseCase` to persist the token after logging in. We will need a new usecase to check if the user has logged in or not for navigating in the next integration task https://github.com/luongvo/kmm-survey/pull/52/files#diff-276576bec5de7fffde347457a897fece6ba7401740667fb1acc68967f808896d.